src/sentry/api/bases/organization_events.py

```diff
@@ -456,6 +456,9 @@ def get_rollup(
             )
         # If the user sends an invalid interval, use the default instead
         except InvalidSearchQuery:
+            # on RPC don't use default
+            if use_rpc:
+                raise
```

Comment:
What does this mean?

Reply:
Instead of catching the InvalidSearchQuery, it just reraises it on rpc. On old queries the functionality is to use the default interval the frontend would've applied